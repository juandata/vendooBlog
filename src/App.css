.App {
    margin: 33px 25px 33px 20px;
}
.MainHeader {
    display: flex;
    align-items: baseline;
    margin-bottom: 47px;
    
}


