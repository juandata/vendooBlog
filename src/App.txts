import React from "react";
import "./NavBar.css";
const App = props => <nav className="NavBar">Latest Updates</nav>;

export default App;
