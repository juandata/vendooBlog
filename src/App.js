import React from "react";

import MainTitle from "./MainTitle";
import NavBar from "./NavBar";
import Thumbnail from "./Thumbnail";
import Button from "./Button";

import snowMountainImg from './assets/snowMountains.png';
import bigCitiesImg from './assets/bigCities.png';
import longDesertImg from './assets/longDesert.png'
import deepOceanImg from './assets/deepOceans.png';
import buttonBackgroundImg from './assets/buttonBackground.svg';
import './App.css';

export default function App() {
  return (
    <div className="App">
      <header className="MainHeader">
        <MainTitle />  <NavBar />

      </header>
      <main className="MainSection">
        <section className="Section1">
        <Thumbnail Title="Snow Mountains" Likes={10} Date={"12/18/90"} Img={snowMountainImg} />
     
          <div className="InnerSection">
          <Thumbnail Title="Big Cities" Likes={10}  Img={bigCitiesImg}  />
        <Thumbnail Title="Long Desert" Likes={1}   Img={longDesertImg} />
          </div>
        </section>
        <section className="Section2">
        <Thumbnail Title="Long Desert" Likes={10} Date={"6/6/96"} Img={deepOceanImg} Id="ThumbnailSection2" />
        <Button Img={buttonBackgroundImg} />

        </section>
      </main> 
    </div>
  );
}




/**  <header className="MainHeader">
        </header>
        <main className="MainSection">
          <section className="Section1">

          </section>
          <section className="Section2">

          </section>

        </main> */