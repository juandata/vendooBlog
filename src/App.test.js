import React from 'react';
import { shallow, mount } from 'enzyme';
import App from './App';


it('App should render correctly', () => {
    const component = shallow(<App/>);
    expect(component).toMatchSnapshot();
  });

  it('App should render correctly with mount', () => {
    const component = mount(<App/>);
    expect(component).toMatchSnapshot();
    component.unmount()
  });