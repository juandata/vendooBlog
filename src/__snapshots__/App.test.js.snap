// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="App"
>
  <header
    className="MainHeader"
  >
    <MainTitle />
      
    <NavBar />
  </header>
  <main
    className="MainSection"
  >
    <section
      className="Section1"
    >
      <Thumbnail
        Date="12/18/90"
        Img="snowMountains.png"
        Likes={10}
        Title="Snow Mountains"
      />
      <div
        className="InnerSection"
      >
        <Thumbnail
          Img="bigCities.png"
          Likes={10}
          Title="Big Cities"
        />
        <Thumbnail
          Img="longDesert.png"
          Likes={1}
          Title="Long Desert"
        />
      </div>
    </section>
    <section
      className="Section2"
    >
      <Thumbnail
        Date="6/6/96"
        Id="ThumbnailSection2"
        Img="deepOceans.png"
        Likes={10}
        Title="Long Desert"
      />
      <Button
        Img="buttonBackground.svg"
      />
    </section>
  </main>
</div>
`;

exports[`App should render correctly with mount 1`] = `
<App>
  <div
    className="App"
  >
    <header
      className="MainHeader"
    >
      <MainTitle>
        <div
          className="MainTitle"
        >
          Blog
        </div>
      </MainTitle>
        
      <NavBar>
        <nav
          className="NavBar"
        >
          Latest Updates
        </nav>
      </NavBar>
    </header>
    <main
      className="MainSection"
    >
      <section
        className="Section1"
      >
        <Thumbnail
          Date="12/18/90"
          Img="snowMountains.png"
          Likes={10}
          Title="Snow Mountains"
        >
          <article
            className="ArticleWrapper"
            style={
              Object {
                "backgroundImage": "url(snowMountains.png)",
              }
            }
          >
            <div
              className="PicturesAllInfoWrapper"
            >
              <header
                className="ArticleHeader"
              >
                Snow Mountains
              </header>
              <div
                className="PicturesInfoWrapper"
              >
                <p
                  className="LikesInfo"
                >
                  10
                   Likes
                </p>
                <p
                  className="DateInfo"
                >
                  12/18/90
                </p>
              </div>
            </div>
          </article>
        </Thumbnail>
        <div
          className="InnerSection"
        >
          <Thumbnail
            Img="bigCities.png"
            Likes={10}
            Title="Big Cities"
          >
            <article
              className="ArticleWrapper SmallThumbnail"
              style={
                Object {
                  "backgroundImage": "url(bigCities.png)",
                }
              }
            >
              <div
                className="PicturesAllInfoWrapper"
              >
                <header
                  className="ArticleHeader SmallFont"
                >
                  Big Cities
                </header>
                <div
                  className="PicturesInfoWrapper"
                >
                  <p
                    className="LikesInfo"
                  >
                    10
                     Likes
                  </p>
                </div>
              </div>
            </article>
          </Thumbnail>
          <Thumbnail
            Img="longDesert.png"
            Likes={1}
            Title="Long Desert"
          >
            <article
              className="ArticleWrapper SmallThumbnail"
              style={
                Object {
                  "backgroundImage": "url(longDesert.png)",
                }
              }
            >
              <div
                className="PicturesAllInfoWrapper"
              >
                <header
                  className="ArticleHeader SmallFont"
                >
                  Long Desert
                </header>
                <div
                  className="PicturesInfoWrapper"
                >
                  <p
                    className="LikesInfo"
                  >
                    1
                     Likes
                  </p>
                </div>
              </div>
            </article>
          </Thumbnail>
        </div>
      </section>
      <section
        className="Section2"
      >
        <Thumbnail
          Date="6/6/96"
          Id="ThumbnailSection2"
          Img="deepOceans.png"
          Likes={10}
          Title="Long Desert"
        >
          <article
            className="ArticleWrapper ThumbnailSection2"
            style={
              Object {
                "backgroundImage": "url(deepOceans.png)",
              }
            }
          >
            <div
              className="PicturesAllInfoWrapper"
            >
              <header
                className="ArticleHeader"
              >
                Long Desert
              </header>
              <div
                className="PicturesInfoWrapper"
              >
                <p
                  className="LikesInfo"
                >
                  10
                   Likes
                </p>
                <p
                  className="DateInfo"
                >
                  6/6/96
                </p>
              </div>
            </div>
          </article>
        </Thumbnail>
        <Button
          Img="buttonBackground.svg"
        >
          <button
            className="Button"
            style={
              Object {
                "backgroundImage": "url(buttonBackground.svg)",
              }
            }
          >
            <div
              className="ButtonEffect"
            >
              <div
                className="ButtonInner"
              >
                <div>
                  View More
                </div>
              </div>
            </div>
          </button>
        </Button>
      </section>
    </main>
  </div>
</App>
`;
